### PR TITLE
#26 pr/change tests dotnet core

### PR DESCRIPTION
**Pull request by jhgoodwin**
I think this fixes the suggestion I made for #26 

I basically moved the tests into its own project for Moq.DataContext since this library is not .net core friendly.

This would allow me to write .net core code and keep it in sync with your library.

### CODE REVIEW

File: Moq.Dapper.Test/Moq.Dapper.Test.csproj [hunk 1]
```diff
@@ -1,82 +1,34 @@
...
-  </ItemGroup>
   <ItemGroup>
-    <Compile Include="DapperExecuteAsyncTest.cs" />
-    <Compile Include="DapperExecuteTest.cs" />
-    <Compile Include="DapperQueryAsyncTest.cs" />
-    <Compile Include="DapperQueryTest.cs" />
-    <Compile Include="DapperTest.cs" />
-    <Compile Include="DataContextTest.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
+    <PackageReference Include="Dapper" Version="1.50.5" />
```
Comment (UnoSD) [inline Moq.Dapper.Test/Moq.Dapper.Test.csproj]: Can you please consolidate the package versions? Other projects run on 1.50.2. We could update all to the latest (Also for other packages).

File: Moq.Dapper.Test/Moq.Dapper.Test.csproj [hunk 2]
```diff
@@ -1,82 +1,34 @@
...
   <ItemGroup>
-    <ProjectReference Include="..\Moq.Dapper\Moq.Dapper.csproj">
-      <Project>{7430287c-81cf-4bb0-889a-d73aaa7e8313}</Project>
-      <Name>Moq.Dapper</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\Moq.DataContext\Moq.DataContext.csproj">
-      <Project>{ef1dff93-6c0b-4adb-891a-c2c6a1e17ab0}</Project>
-      <Name>Moq.DataContext</Name>
-    </ProjectReference>
+    <Folder Include="Data" />
```
Comment (UnoSD) [inline Moq.Dapper.Test/Moq.Dapper.Test.csproj]: This folder seems empty, was it added intentionally?

**Comment by UnoSD** [comment]
Hi John, I tried checking out and compiling the change, but I get `Ambiguous project name 'Moq.Dapper'.` from the package manager on restore; maybe something on my machine, but if it happens to everyone I may need to look into it before merging the PR. Thanks. Stef.

**Comment by jhgoodwin** [comment]
I'll check it out on a windows machine. I didn't have framework, but I'll address these issues.

**Comment by UnoSD** [comment]
Thank you

**Comment by UnoSD** [comment]
After only one year I managed to merge this :) thanks @jhgoodwin

**Pull request merged**